### PR TITLE
Please no warnings

### PR DESCRIPTION
**Pull request by plaindocs**

### CODE REVIEW

File: docs/conf/portland/2018/index.rst [hunk 1]
```diff
@@ -1,6 +1,7 @@
 :template: 2018/na/index.html
 
-.. title:: Home - Write the Docs Portland 2018
+Home - Write the Docs Portland 2018
```
Comment (ericholscher) [inline docs/conf/portland/2018/index.rst]: Did this not blow up on rendering? There's a reason I didn't use this solution, but we _could_ easily make them all as orphans.

**Comment by ericholscher** [comment]
+1 on just marking all of them as orphan's for now. Not sure why that didn't occur to me :sigh:

File: docs/conf/portland/2018/index.rst (hunk 1 above)
Comment (plaindocs) [reply docs/conf/portland/2018/index.rst]: As long as we don't use the title in the body of the index page template, should be fine right?

Comment (plaindocs) [reply docs/conf/portland/2018/index.rst]: I took a quick glance and it seemed ok

**Comment by plaindocs** [comment]
Right, *instead* of adding them to the toc.

**Pull request merged**